src/kv/replicator.h

```diff
@@ -191,6 +76,35 @@ namespace kv
     {
       return true;
     }
+
+    bool on_request(kv::TxHistory::RequestCallbackArgs args) override
```

Comment:
As per my other comment, this should go as the `StubReplicator` should only derive from `kv::Replicator`.